### ISSUE TITLE
Order API returns additional_information on payment without array keys. 

### ISSUE THREAD

**Issue by drake1701** [magento/magento2#13738]
### Preconditions

1. Magento 2.2.0
2. PHP 7.0.22
3. SOAP API with token admin authentication

### Steps to reproduce

1. I'm using SoapClient from a standalone PHP script to access the API.
2. Client is using ASP through the REST interface, results are the same.

### Expected result

The <additionalInformation> element in the <payment> element should have values identified by tag.
```
<additionalInformation>
    <save>0</save>
    <acceptjs_key/>
    <acceptjs_value/>
    <method_title>Credit Card</method_title>
    <response_code>1</response_code>
...
```

### Actual result

The values return simply in <item> tags, making them fairly useless.
```
<additionalInformation>
    <item>0</item>
    <item/>
    <item/>
    <item>Credit Card</item>
    <item>1</item>
```

**Comment by koenner01** [comment]
How are you fetching the payment object ? Have you tried using a \Magento\Framework\Webapi\ServiceOutputProcessor to process to object into its data model?

Example:
```
$formattedData = $this->serviceOutputProcessor->process(
    $order,
    'Magento\Sales\Api\OrderRepositoryInterface',
    'get'
);
```

**Comment by magento-engcom-team** [comment]
Not a bug according to http://devdocs.magento.com/swagger/#!/salesOrderRepositoryV1/salesOrderRepositoryV1GetGet


![docs](https://user-images.githubusercontent.com/31669971/36430226-508be248-165d-11e8-9bf6-d9582218e1e7.png)

**Comment by magento-engcom-team** [comment]
@drake1701, thank you for your report.
This seems to be correct Magento behavior. Please refer to the [Community Forums](https://community.magento.com/) or the [Magento Stack Exchange](https://magento.stackexchange.com/) site for advice or general discussion about this.
Otherwise you may submit Pull Request with the suggested changes.

**Comment by rodrigowebjump** [comment]
Hi @drake1701 This was fixed in Magento 2.3.1 including the key/values in extension attributes

https://github.com/magento/magento2/commit/c0b52afb5f365e0f1239e0fff5a9b964b01953da
